examples/amp-render-resize-to-contents.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
Storybook?

Reply:
I mean instead of creating a new file in `examples/`

Reply:
These are examples I've created for testing this features and are generally useful for @morsssss and @CrystalOnScript.

Reply:
Based on my (brief) chat with @morsssss, they do look at these examples, however might simplify them for amp.dev. They're not aware of storybooks though.